frontend/src/pages/dashboard/Dashboard.tsx

```diff
@@ -77,7 +77,7 @@ const Dashboard = () => {
                 <Tabs
                   tabs={tabs}
                   changeTab={(value: unknown) => {
-                    searchParams.set(Filters.AVAILABLE_STATUS, 'ALL');
+                    searchParams.delete(Filters.AVAILABLE_STATUS);
```

Comment:
Deleting instead of setting - same outcome, fewer params passed into the api request